ci_framework/roles/set_openstack_containers/templates/containers.j2

```diff
@@ -0,0 +1,91 @@
+
+# Ironic
+IRONIC_API_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-ironic-api:{{ cifmw_set_openstack_containers_tag }}
+IRONIC_CONDUCTOR_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-ironic-conductor:{{ cifmw_set_openstack_containers_tag }}
+IRONIC_INSPECTOR_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-ironic-inspector:{{ cifmw_set_openstack_containers_tag }}
+IRONIC_PXE_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-ironic-pxe:{{ cifmw_set_openstack_containers_tag }}
+IRONIC_NEUTRON_AGENT_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-ironic-neutron-agent:{{ cifmw_set_openstack_containers_tag }}
+
+# Horizon
+HORIZON_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-horizon:{{ cifmw_set_openstack_containers_tag }}
+
+# Heat
+HEAT_API_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-heat-api:{{ cifmw_set_openstack_containers_tag }}
+HEAT_CFNAPI_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-heat-api-cfn:{{ cifmw_set_openstack_containers_tag }}
+HEAT_ENGINE_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-heat-engine:{{ cifmw_set_openstack_containers_tag }}
+
+# Telemetry
+CEILOMETER_CENTRAL_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-ceilometer-central:{{ cifmw_set_openstack_containers_tag }}
+CEILOMETER_CENTRAL_INIT_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-ceilometer-central:{{ cifmw_set_openstack_containers_tag }}
+CEILOMETER_COMPUTE_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-ceilometer-compute:{{ cifmw_set_openstack_containers_tag }}
+CEILOMETER_COMPUTE_INIT_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-ceilometer-compute:{{ cifmw_set_openstack_containers_tag }}
+CEILOMETER_NOTIFICATION_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-ceilometer-notification:{{ cifmw_set_openstack_containers_tag }}
+CEILOMETER_SGCORE_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_sg_core_image }}
+TELEMETRY_NODE_EXPORTER_IMAGE_URL_DEFAULT: {{ cifmw_set_openstack_containers_node_exporter_image }}
```

Comment:
This seems like a lot to keep up with as the containers are added/removed/renamed? Is there any way to read/inherit this list from elsewhere?